oep-template.rst

```diff
@@ -0,0 +1,108 @@
+=================================
+OEP-π: OEP Template
+=================================
+
+.. This is a template that can be used when starting a new OEP
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-π`                              |
++---------------+-------------------------------------------+
+| Title         | <OEP title>                               |
++---------------+-------------------------------------------+
+| Last-Modified | <date string>                             |
++---------------+-------------------------------------------+
+| Author        | <list of authors' real names and          |
+|               | optionally, email addrs>                  |
++---------------+-------------------------------------------+
+| Arbiter       | <OEP czar's real name>                    |
++---------------+-------------------------------------------+
+| Status        | <Draft | Accepted | Deferred |            |
+|               | Rejected | Withdrawn | Final |            |
+|               | Replaced>                                 |
++---------------+-------------------------------------------+
+| Type          | <Architecture | Best Practice |           |
+|               | Process>                                  |
++---------------+-------------------------------------------+
+|  Created      | <date created on, in yyyy-mm-dd format>   |
++---------------+-------------------------------------------+
+| `Replaces`    | <OEP number>                              |
++---------------+-------------------------------------------+
+| `Replaced-By` | <OEP number>                              |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+A short (~200 word) description of the technical issue being addressed.
+
+
+Specification
+=============
+
+The technical specification should describe the syntax and semantics of any new API,
+or the details of what the Best Practice, Process, or Archictecture being proposed
+by the OEP are.
+
+
+Motivation
+==========
+
+The motivation is critical for OEPs that want to change Open edX. It should
+clearly explain why the existing achictecture or process is inadequate to
+address the problem that the OEP solves, or why Open edX would be significantly
+improved by adopting the best practice. OEP submissions without sufficient
+motivation may be rejected outright.
```

Comment:
> OEP submissions without sufficient motivation may be rejected outright.

Could we soften this a bit (or remove it)? I think it makes the process feel less approachable.